packages/template-block/src/TemplateBlock.spec.ct.tsx

```diff
@@ -56,6 +56,15 @@ const getTemplateDummyWithPages = () => {
     return templateDummy;
 };
 
+const getValueFromCssVariable = (cssVarName: string, styleName: string) => {
+    const dummy = document.createElement('span');
+    dummy.style.setProperty(styleName, `var(${cssVarName})`);
+    document.body.appendChild(dummy);
+    const value = window.getComputedStyle(dummy).getPropertyValue(styleName).trim();
+    dummy.remove();
+    return value;
+};
```

Comment:
Yes, this is a bit wild, and the values for secondary and primary in this case are actually the same. However, imho it's actually testing the applied values.. Or would you sth like `data-text-button-style="secondary"` to the DOM element that can be tested?

Reply:
Would just test if the button styles are applied.

So if you wanna test `font-size` you just check that the value is `var(--f-theme-settings-button-primary-font-size)` and not the value of the var

Reply:
Doesn't work, it only gets the actual value rather than the variable itself. I tried to search how to get the variable as a string, but didn't find anything. So I'm applying the same variable and comparing the outcome basically.

![image](https://github.com/Frontify/guideline-blocks/assets/9266535/1ece105c-8af1-49da-b561-1dac49a374d6)

Reply:
Damn that's genious, can't believe I haven't found this approach 🙈 Keep on learning 🚀 

Fixed in #[49fc2ed](https://github.com/Frontify/guideline-blocks/pull/761/commits/49fc2ed2bc9dc54406c68ed8fda6d1e4c7f0dd06)